wagtail/admin/templates/wagtailadmin/admin_base.html

```diff
@@ -1,20 +1,18 @@
-{% extends "wagtailadmin/skeleton.html" %}
-{% load i18n wagtailadmin_tags %}
-
-{% block css %}
-    <link rel="stylesheet" href="{% versioned_static 'wagtailadmin/css/core.css' %}">
-
-    {% hook_output 'insert_global_admin_css' %}
-    {% hook_output 'insert_editor_css' %}
-
-    {% block extra_css %}{% endblock %}
-{% endblock %}
-
-{% block branding_favicon %}
-    <link rel="shortcut icon" href="{% versioned_static 'wagtailadmin/images/favicon.ico' %}" />
-{% endblock %}
-
-{% block js %}
+{% extends "wagtailadmin/skeleton.html" %} {% load i18n wagtailadmin_tags %} {%
```

Comment:
Something that's gone wrong with your editor by the looks of things, the only change in this file should be the removal one one line. 

It looks like this file has had large changes for reformatting. 

Maybe run `make format-server` or revert the partial commit.